### PR TITLE
Make System Broker Prow job mandatory and add trigger for scripts folder change

### PR DESCRIPTION
**Pull request by danielgospodinow**
**Description**

Changes proposed in this pull request:

- Make System Broker Prow job mandatory
- Add scripts folder change trigger

### CODE REVIEW

**Comment by NickyMateev** [comment]
**Do not merge until System Broker is merged in Compass repo master branch!**

**Review by desislavaa — APPROVED** [review]

**Pull request merged**
